pyfakefs/fake_filesystem.py

```diff
@@ -2915,6 +2918,19 @@ def _add_standard_streams(self) -> None:
         self._add_open_file(StandardStreamWrapper(sys.stdout))
         self._add_open_file(StandardStreamWrapper(sys.stderr))
 
+    def _create_temp_dir(self):
+        # the temp directory is assumed to exist at least in `tempfile`,
+        # so we create it here for convenience
+        temp_dir = tempfile.gettempdir()
+        if not self.exists(temp_dir):
+            self.create_dir(temp_dir)
+        if sys.platform != "win32" and not self.exists("/tmp"):
+            # under Posix, we also create a link in /tmp if the path does not exist
+            self.create_symlink("/tmp", temp_dir)
```

Comment:
See https://docs.python.org/3/library/tempfile.html#tempfile.gettempdir
The possible values of tempfile.gettempdir() would be '/tmp', '/var/tmp', or '/usr/tmp' and I would think that if either '/var/tmp' or '/usr/tmp' were created, then a symlink '/tmp' would not be part of the native file system.

Reply:
I still don't understand why this added symlink is required.